backend/api/Startup.cs

```diff
@@ -158,10 +158,10 @@ public void ConfigureServices(IServiceCollection services)
                 })
                 .AddJwtBearer(options =>
                 {
-                    var key = Encoding.ASCII.GetBytes(Configuration["Keycloak:Secret"]);
+                    var key = Encoding.ASCII.GetBytes(this.Configuration["Keycloak:ServiceAccount:Secret"]);
```

Comment:
Looks like you're mixing client types in keycloak and the api.  You shouldn't need a secret to validate the JWT that was generated by the frontend app.

Reply:
Thank you for this catch, I've fixed this oversight